coldfront/core/project/tests/test_commands/test_get_survey_responses.py

```diff
@@ -0,0 +1,100 @@
+from django.test import TestCase
+from coldfront.core.project.models import *
+from django.contrib.auth.models import User
+from django.core.management import call_command
+
+
+from coldfront.core.project.models import SavioProjectAllocationRequest, Project, ProjectAllocationRequestStatusChoice, ProjectStatusChoice, FieldOfScience, User
+
+from io import StringIO
+import sys
+import json
+from csv import DictReader
+
+
+class TestGetSurveyResponses(TestCase):
+    """
+    Base Class for testing
+    """
+
+    def setUp(self):
+        # create dummy survey responses
+        fixtures = []
+        filtered_fixtures = []
+
+        # dummy params
+        allocation_type = SavioProjectAllocationRequest.FCA
+        pool = False
+
+        for index in range(5):
+            pi = User.objects.create(
+                username=f'test_user{index}', first_name='Test', last_name='User', is_superuser=True)
+            project_prefix = 'fc_' if index % 2 else ''
+            project = Project.objects.create(name=f'{project_prefix}test_project{index}', status=ProjectStatusChoice.objects.create(
```

Comment:
For consistency/convenience, I'd recommend running the commands that generate the expected default objects so you don't have to.

- See `test_pending_auto_request_reminder.setUp` for an example.
- `add_default_project_choices` will create the default `ProjectStatusChoice` and `ProjectAllocationRequestStatusChoice` objects.
- Other commands in different modules, if needed, will do the same for other needed defaults.
- You can refer to the "Django management commands" section of the Ansible playbook `bootstrap/development/playbook.yml` to see the full list of commands that are run during setup.